### PR TITLE
tests: add and follow solium rules

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,8 @@
   "scripts": {
     "test": "npx truffle test",
     "build": "bash scripts/build.sh",
-    "deploy": "bash scripts/deploy.sh",
-    "copy-artifacts": "bash scripts/copy-artifacts.sh",
-    "app": "bash scripts/app.sh"
```
Comment (nventuro) [inline package.json]: Could you restore these scripts?

Comment (elopio) [reply package.json]: Done. Thanks for catching it.

**Comment by nventuro** [comment]
Neat! Could you fix the merge conflict and squash the `fix merge` commits? Thanks!

**Comment by elopio** [comment]
Last iteration, hopefully. :crossed_fingers:

**Pull request merged**